war/src/main/webapp/WEB-INF/jonas-web.xml

```diff
@@ -25,7 +25,6 @@ THE SOFTWARE.
 
 <jonas-web-app xmlns="http://www.objectweb.org/jonas/ns"
         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-        xsi:schemaLocation="http://www.objectweb.org/jonas/ns
-        http://www.objectweb.org/jonas/ns/jonas-web-app_4_0.xsd" >
+        xsi:schemaLocation="http://www.objectweb.org/jonas/ns/jonas-web-app_4_0.xsd" >
```

Comment:
:-1: this part has a special syntax and is *not* a mistake. Each "entry" is a pair. The first is actually a namespace name (`http://` is misleading here, IIRC `urn://` might be recommended to clarify it's a logical thing, but I'm not an XML expert), and the second string is the actual location.

Also, a good practice is to file two different PR for the things here. Or at the very least in two different commits. Keep things grouped by "concern" in general.

Keep it coming @arshdkhn1 ! Way to go!

#GSoC